UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -68,67 +73,95 @@ private void OnValidate()
 
 	private void OnLevelFinishedLoading(Scene scene, LoadSceneMode mode)
 	{
-		GetRoundTime = RoundTime;
+        GetRoundTime = RoundTime;
+        GetShuttleTime = ShuttleTime;
 	}
 
-	public void SyncTime(float currentTime)
-	{
-		if (!CustomNetworkManager.Instance._isServer)
-		{
-			GetRoundTime = currentTime;
-			if (currentTime > 0f)
-			{
-				counting = true;
-			}
-		}
-	}
+    public void SyncTime(float currentTime)
+    {
+        Debug.Log("TRACE: SyncTime " + currentTime);
+
+        PostToChatMessage.Send("TRACE: SyncTime " + currentTime, ChatChannel.OOC);
+
+        if (!CustomNetworkManager.Instance._isServer)
+        {
+            GetRoundTime = currentTime;
+            if (currentTime > 0f)
+            {
+                counting = true;
+                ShuttleCounting = false;
+            }
+        }
+    }
 
-	public void ResetRoundTime()
+    public void ResetRoundTime()
 	{
 		GetRoundTime = RoundTime;
-		waitForRestart = false;
+        GetShuttleTime = ShuttleTime;
 		counting = true;
-		restartTime = 10f;
+        ShuttleCounting = false;
 		UpdateRoundTimeMessage.Send(GetRoundTime);
 	}
 
-	private void Update()
-	{
-		if (waitForRestart)
-		{
+    public void ReportScores()
+    {
+        foreach (ConnectedPlayer player in PlayerList.Instance.InGamePlayers)
+        {
+            if (GetComponent<HealthBehaviour>().IsDead == false)
+            {
+                LifeCount++;
+                PostToChatMessage.Send(player.Name + " has survived the Syndicate terrorist attack on the NSS Cyberiad.", ChatChannel.OOC);
+            }
+            if (LifeCount == 0)
+            {
+                PostToChatMessage.Send("No one has survived the Syndicate terrorist attack on the NSS Cyberiad.", ChatChannel.OOC);
+                PostToChatMessage.Send("The nuke ops have killed everyone and won.", ChatChannel.OOC);
+            }
+            if (LifeCount == 1)
+            {
+                PostToChatMessage.Send(LifeCount + " person survived the Syndicate terrorist attack on the NSS Cyberiad.", ChatChannel.OOC);
+                PostToChatMessage.Send("The nuke ops have failed to detonated the bomb.", ChatChannel.OOC);
+            }
+            if (LifeCount > 1)
+            {
+                PostToChatMessage.Send(LifeCount + " people survived the Syndicate terrorist attack on the NSS Cyberiad.", ChatChannel.OOC);
+                PostToChatMessage.Send("The nuke ops have failed to detonated the bomb.", ChatChannel.OOC);
+            }
```

Comment:
Repetitive text should be saved in variable / field to avoid future inconsistencies